cpp/tests/io/comp/decomp_test.cpp

```diff
@@ -14,15 +14,16 @@
  * limitations under the License.
  */
 
-#include <io/comp/gpuinflate.hpp>
-#include <io/utilities/hostdevice_vector.hpp>
-#include <src/io/comp/nvcomp_adapter.hpp>
-
-#include <cudf/utilities/default_stream.hpp>
+// TODO : don't include src from tests!
```

Comment:
Adding a comment to track this TODO. It might be easiest to solve this in a separate PR, but I'm not 100% sure what the outcome should be. There seem to be quite a few test files, especially I/O tests, that rely on files from `src`.